tests/transforms/augmentation/test_random_simulation.py

```diff
@@ -0,0 +1,29 @@
+import torch
+from torchio.transforms import RandomSimulation
+from torchio import DATA
+from ...utils import TorchioTestCase
+
+
+class TestRandomSimulation(TorchioTestCase):
+    """Tests for `RandomSimulation`."""
+    def test_random_simulation(self):
+        transform = RandomSimulation()
+        transformed = transform(self.sample)
+        self.assertIn('image', transformed)
+
+    def test_deterministic_simulation(self):
```

Comment:
As you can see in the logs, this test is failing: https://travis-ci.org/github/fepegar/torchio/jobs/706568942

Reply:
My bad, I changed the Gaussian sampling without running the tests again.